elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideDynamicConfiguration.java

```
@@ -0,0 +1,148 @@
+            puiProps.putAll(puiPropertyMap);
+
+            //Create Elide dynamic Persistence Unit
+            ElideDynamicPersistenceUnit elideDynamicPersistenceUnit =
+                    new ElideDynamicPersistenceUnit("dynamic", compiler.classNames, puiProps,
+                    compiler.getClassLoader());
+            elideDynamicPersistenceUnit.setNonJtaDataSource(source);
+            elideDynamicPersistenceUnit.setJtaDataSource(source);
+
+            HibernateJpaVendorAdapter vendorAdapter = new HibernateJpaVendorAdapter();
+            vendorAdapter.setShowSql(jpaProperties.isShowSql());
+            vendorAdapter.setGenerateDdl(jpaProperties.isGenerateDdl());
+            if (jpaProperties.getDatabase() != null) {
+                vendorAdapter.setDatabase(jpaProperties.getDatabase());
+            }
+            if (jpaProperties.getDatabasePlatform() != null) {
+                vendorAdapter.setDatabasePlatform(jpaProperties.getDatabasePlatform());
+            }
+
+            LocalContainerEntityManagerFactoryBean bean = new LocalContainerEntityManagerFactoryBean();
+            bean.setJpaVendorAdapter(vendorAdapter);
+
+            //Add JPA Properties from Application.yaml
+            bean.setJpaPropertyMap(jpaPropMap);
+
+            //Add Classes
+            bean.setJpaPropertyMap(puiPropertyMap);
+
+            bean.setPersistenceUnitManager(new PersistenceUnitManager() {
+                @Override
+                public PersistenceUnitInfo obtainDefaultPersistenceUnitInfo() throws IllegalStateException {
+                    return elideDynamicPersistenceUnit;
+                }
+
+                @Override
+                public PersistenceUnitInfo obtainPersistenceUnitInfo(String persistenceUnitName)
+                        throws IllegalArgumentException, IllegalStateException {
+                    return elideDynamicPersistenceUnit;
+                }
+            });
+
+            return bean;
+        } catch (Exception e) {
```

Two things - let's not catch Exception here (it is too broad).  Also, if there is an error, we'll want to bring down the service.  Let the exceptions propagate.